airbyte-integrations/connectors/source-freshservice/source_freshservice/schemas/satisfaction_survey_responses.json

```diff
@@ -0,0 +1,69 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": "integer"
+    },
+    "overall_rating": {
+      "type": "integer"
+    },
+    "overall_rating_text": {
+      "type": "string"
+    },
+    "primary_question": {
+      "type": "string"
+    },
+    "questionnaire_responses": {
+      "type": "object",
+      "items": {
+        "type": "object",
+        "additionalProperties": true,
+        "properties": {
+          "question": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+              "question_text": {
+                "type": "string"
+              }
+            },
+            "required": ["question_text"]
+          },
+          "answers": {
+            "type": "array",
+            "items": [
+              {
+                "type": "object",
+                "additionalProperties": true,
+                "properties": {
+                  "answer_text": {
+                    "type": "string"
+                  }
+                },
+                "required": ["answer_text"]
+              }
+            ]
+          }
+        },
+        "required": ["question", "answers"]
+      }
+    },
+    "created_at": {
+      "type": "string"
```

Comment:
Please follow: https://docs.airbyte.com/connector-development/schema-reference
Always or the std is to use `["null", "string"]` Review all fields.